features/admin_login_and_update_menu.feature

```diff
@@ -25,4 +29,16 @@ Feature: Admin Log in
 
     Scenario: Owner can log in to the system
         When owner has logged in
-        Then I see "Welcome owner!"
\ No newline at end of file
+        Then I see "Welcome owner!"
+
+    Scenario: Owner can uppdate update menu
+        When owner has logged in
+        And I see "Welcome owner!"
+        And I click "Edit Starter"
+        And I fill in "product[name]" with "Tomato salad"
```

Comment:
I would try to use:
```
 And I fill in "Name" with "Tomato salad"
```
if you format your `form_with` properly, Capybara should be able to identify the field by its label.

Reply:
That goes for next 2 lines as well.